src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -447,6 +447,143 @@ JSON example:
+| IntendedFor          | REQUIRED              | [string][] or [array][] of [strings][] | One or more filenames with paths relative to the subject subfolder, with forward slashes, referring to ASL timeseries for which the m0scan is intended.                                                                                                                                                                                                                           |
+| AcquisitionVoxelSize | RECOMMENDED           | [array][] of [numbers][]               | An array of numbers with a length of 3, in millimeters. This parameter denotes the original acquisition voxel size, excluding any inter-slice gaps and before any interpolation or resampling within reconstruction or image processing. Any point spread function effects, for example due to T2-blurring, that would decrease the effective resolution are not considered here. |
+
+The following table recapitulates the ASL field dependencies. If Source field (column 1) contains the Value specified in column 2, then the Requirements in column 4 are imposed on the Dependent fields in column 3.
+
+| **Source field**              | **Value**                       | **Dependent field**           | **Requirements**              |
+|-------------------------------|---------------------------------|-------------------------------|-------------------------------|
+| PulseSequenceType             | 2D acquisition / 3D acquisition | SliceTiming                   | [X] / []                      |
+| LookLocker                    | true                            | FlipAngle                     | [X]                           |
+| M0                            | true / false                    | `*_aslcontext.tsv`            | Contains `m0scan` / `control` |
+| LabelingType                  | PASL                            | BolusCutOffFlag               | [X]                           |
+| BolusCutOffFlag               | true / false                    | BolusCutOffDelayTime          | [X] / []                      |
+| BolusCutOffFlag               | true / false                    | BolusCutOffTechnique          | [X] / []                      |
+| `*_aslcontext.tsv`            | cbf                             | Units                         | [X]                           |
+| RepetitionTime / VolumeTiming | [X] / []                        | VolumeTiming / RepetitionTime | [] / [X]                      |
+| VolumeTiming                  | [X]                             | AcquisitionDuration           | [X]                           |
```

Comment:
this table is problematic in terms of markdown syntax. the square brackets need to either be escaped using backslashes like `\[ X \]`, or they need to be wrapped in code syntax using backticks:

```
`[X]`
```

Reply:
Ah it might indeed be this one. I'll try with your tips. Thanks

Reply:
yes, you last commit 9f752fc fixed the Travis linter! :-)

Reply:
YEs thanks! I was confused as the line reference was confusing. Anyway, learned a lot today!

Reply:
Stefan, still problems with 2 tables in the readthedocs. Any ideas?

Reply:
> Stefan, still problems with 2 tables in the readthedocs. Any ideas?

Can you provide some links and/or screenshots, please? :)

Reply:
https://bids-specification--652.org.readthedocs.build/en/652/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#pcasl-specific-metadata-fields

![image](https://user-images.githubusercontent.com/41481345/97172924-0ade4380-1790-11eb-99e5-64c23977b09c.png)
![image](https://user-images.githubusercontent.com/41481345/97172954-1af62300-1790-11eb-8046-0b4024475a31.png)

Reply:
@patsycle I think you are looking at the wrong "build artifact". The draft gets rebuilt after each commit that you do. This is the most recent one: https://bids-specification--652.org.readthedocs.build/en/652/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#pcasl-specific-metadata-fields

To get the most recent artifact, scroll to the bottom of the pull request user interface, and click on "Details" for the "docs/readthedocs.org:bids-specification" check, see this screenshot:

![image](https://user-images.githubusercontent.com/9084751/97175587-087de880-1794-11eb-87da-7f9081b30f57.png)

Reply:
It seems I was looking at the right one, but the link was still in cache. Using a different browser worked. Thanks for all you help!